projects/organization-management/src/app/shared/user/user-profile-form/user-profile-form.component.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'ish-user-profile-form',
+  templateUrl: './user-profile-form.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
+})
+export class UserProfileFormComponent {
+  @Input() form: FormGroup;
+  @Input() titles: string[];
+  @Input() editing = false;
```

Comment:
parameter is not needed? - you can check the form values if you still need a differentiation between edit and new